salesagility/SuiteCRM
# Feature/pwdsecure 2 - fix password expires issue

## Description



This change corrects the issue that a users password expires immediately, which cause by the changes to password management due to the defaults on the config have not been set correctly.
 
## Motivation and Context

- Bug fix

## How To Test This

- Install and log into a user
- You password should only expire when it has expired

## Types of changes


### Final checklist
